nifi-assembly/pom.xml

```diff
@@ -19,6 +19,10 @@ language governing permissions and limitations under the License. -->
     <artifactId>nifi-assembly</artifactId>
     <packaging>pom</packaging>
     <description>This is the assembly Apache NiFi</description>
+    <!--Add a blank run.as in case RPM isn't being used -->
+    <properties>
+        <nifi.run.as />
```

Comment:
I find it overkill (the person would be voluntarily building the RPM) but doable. 

Will adjust the PR